framework/logging.cpp

```diff
@@ -1249,7 +1251,7 @@ void log_platform_message(const char *fmt, ...)
 #undef log_message_skip
 void log_message_skip(int thread_num, SkipCategory category, const char *fmt, ...)
 {
-    if (current_output_format() == SandstoneApplication::OutputFormat::no_output)
+    if (current_output_format() == SandstoneApplication::OutputFormat::no_output || category == DummySkipCategory)
```

Comment:
Will anything be logged if if all threads return SKIP? I expected handling DummySkipCategory somewhere in "postprocessing" phase

Reply:
As long as you don't use DummySkipCategory, the messages will be logged. Only for DummySkipCategory it won't be logged.

If all threads skip, that's a genuine skip right? you will want to use a different log_skip category for that and NOT use DummySkipCategory.